nipype/interfaces/ants/resampling.py

```diff
@@ -349,6 +349,7 @@ class ApplyTransformsInputSpec(ANTSCommandInputSpec):
         "MultiLabel",
         "Gaussian",
         "BSpline",
+        "GenericLabel",
```

Comment:
I think you need to add `traits.Tuple(traits.Str())` to `interpolation_parameters` options, as well.